tests/src/test/scala/cats/tests/FunctionTests.scala

```diff
@@ -25,4 +26,6 @@ class FunctionTests extends CatsSuite {
 
   checkAll("Function1[Int, Int]", ContravariantTests[? => Int].contravariant[Int, Int, Int])
   checkAll("Contravariant[? => Int]", SerializableTests.serializable(Contravariant[? => Int]))
+
+  checkAll("Function1[String, Int]", GroupLaws[Function1[String, Int]].monoid)
```

Comment:
The laws that come algebra [include serializability checks](https://github.com/non/algebra/blob/master/laws/shared/src/main/scala/algebra/laws/GroupLaws.scala#L28), so I don't think we need separate checks for those.